### ISSUE TITLE
missing -r option

### ISSUE THREAD

**Issue by 1moere1** [lgandx/Responder#182]
In the old version of Responder there was a -r option that is missing now. Is there a reason for this?

**Comment by jonyluke** [comment]
It was bypassed in the code, the option was enabled by default, even if -r was not specified

https://twitter.com/PythonResponder/status/1471682291818770434

**Comment by 0xGreen** [comment]
Same! 

responder -h says:
```
Usage: responder -I eth0 -w -r -f
or:
responder -I eth0 -wrf
```
but there is no -r and no -f

Is -f enabled by default in the code as well? if yes, would be better if "usage" is revised. It would also be less confusing :)

**Comment by lgandx** [comment]
Please use git pull in Responder's directory. The latest version doesn't
have that help output.

**Comment by 0xGreen** [comment]
Hmmm,

I was referencing to responder version available in kali, i just cloned it from git and the Usage is correct (without -rf) but the version number is same.

![image](https://user-images.githubusercontent.com/6963584/152090352-1ad1c8ae-613b-4097-86aa-8af2983eec63.png)

The one on the top is the Kali's version and the bottom one is from git.

**Comment by 0xGreen** [comment]
> @0xGreen You're reporting an issue in kali's package not Responder _per se_. Can you close this issue please?

Sure, but i clouldn't see any close issue option. Please feel free to close the issue if you can.

**Comment by antifob** [comment]
@0xGreen Sorry :/ I pinged you instead of 1moere1 Thanks for replying, though. :)

The issue is still relevant to this repository so I would keep it opened. I noticed the `-r` is still mentioned in the README and help message.

A quick git-dissect shows that the flag was removed in https://github.com/antifob/Responder/commit/5d4510cc1d0479b13ece9d58ea60d187daf8cdab and the help message was updated in https://github.com/lgandx/Responder/commit/afb54fa274dc41b04fc3cf14ac43ddf712f82bf7

kali adds a bunch of patches to Responder so their package may be behind and also depending on fast it's made available to end users. https://gitlab.com/kalilinux/packages/responder/
